### ISSUE TITLE
Error clone VM - Too few arguments to function IVirtualBox::createMachine()

### ISSUE THREAD

**Issue by BartekSz95** [BartekSz95/phpvirtualbox#4]
Full description about this issue is on official **phpVirtualBox's** _Issues_ tab:
https://github.com/phpvirtualbox/phpvirtualbox/issues/323

**Comment by BartekSz95** [comment]
Correct error log:
```
Fatal error:  Uncaught ArgumentCountError: Too few arguments to function IVirtualBox::createMachine(), 6 passed in C:\Apache24\htdocs\phpvirtualbox\endpoints\lib\vboxconnector.php on line 1362 and exactly 8 expected in C:\Apache24\htdocs\phpvirtualbox\endpoints\lib\vboxServiceWrappers.php:1545
Stack trace:
#0 C:\Apache24\htdocs\phpvirtualbox\endpoints\lib\vboxconnector.php(1362): IVirtualBox->createMachine('/home/vbox/Virt...', 'VPS2 Clone', NULL, NULL, NULL, false)
#1 C:\Apache24\htdocs\phpvirtualbox\endpoints\lib\vboxconnector.php(952): vboxconnector->remote_machineClone(Array)
#2 C:\Apache24\htdocs\phpvirtualbox\endpoints\api.php(316): vboxconnector->__call('machineClone', Array)
#3 {main}
  thrown in C:\Apache24\htdocs\phpvirtualbox\endpoints\lib\vboxServiceWrappers.php on line 1545
```

**Comment by BartekSz95** [comment]
Fixed on commit [fea1319](https://github.com/BartekSz95/phpvirtualbox/commit/fea1319342660dbc1b8a0a319c62d8af00940fe7)
